File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        vector<int> seen;
        for (const int& i : nums) {
            for (const int& j : seen) {
                if (i == j) {
                    return true;
                }
            }
            seen.push_back(i);
        }
        return false;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        counts = defaultdict(int)
        res = []

        for n in nums:
            counts[n] += 1
        
        res = list(counts.items())
        res.sort(key=lambda x: x[1], reverse=True)
        res = res[0:k]
        return [key for key, val in res]
            

